docs/contrib/docusaurus/_category_.json

```diff
@@ -1,6 +1,6 @@
 {
   "label": "Writing Documentation",
-  "position": -1,
+  "position": 10,
   "link": {
     "type": "generated-index",
     "description": "Guides for the different methods of cooking a Mod and testing a Mod."
```

Comment:
Can you update the category description. I left it as this value on a couple by accident.